pkg/agent/datamodel/mocks.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // CreateMockContainerService returns a mock container service for testing purposes.
-func CreateMockContainerService(containerServiceName, orchestratorVersion string, masterCount, agentCount int, certs bool) *ContainerService {
```

Comment:
CreateMockContainerService is not being used anywhere, removing these from the function should be okay.

Reply:
can we remove the func altogether if it's not being used?

Reply:
Do you think this mock for a container service will be used again? Or does it seem outdated?